pkg/apiserver/storage/testing/watcher_tests.go

```diff
@@ -1481,7 +1480,9 @@ func RunWatchSemantics(ctx context.Context, t *testing.T, store storage.Interfac
 				createdPods = append(createdPods, out)
 			}
 			testCheckResultsInStrictOrder(t, w, scenario.expectedEventsAfterEstablishingWatch(createdPods))
-			testCheckNoMoreResults(t, w)
+
+			// TestEtcdWatchSemantics fails with timeout???
+			//testCheckNoMoreResults(t, w)
```

Comment:
^^^ the last test that is not passing 😢

Reply:
i ran into this one before. are you running it with vscode's `Run Test` button? if i run via the terminal without the 30s timeout flag that vscode adds it passes: 
```
go test -run '^TestEtcdWatchSemantics$' github.com/grafana/grafana/pkg/apiserver/storage/file

ok      github.com/grafana/grafana/pkg/apiserver/storage/file   36.064s
```

Reply:
i removed the comment in https://github.com/grafana/grafana/pull/90382/commits/aa634803d4e3a5d1a7e7e782507f7fadb5b1825f to see if it passes in drone